monai/networks/utils.py

```
@@ -1035,40 +1123,55 @@ def convert_to_trt(
+            else:
+                trt_model = torch.jit.load(filename_or_obj)
 
         with torch.no_grad():
             set_determinism(seed=0)
             torch_out = ensure_tuple(model(*inputs))
             set_determinism(seed=0)
             trt_out = ensure_tuple(trt_model(*inputs))
             set_determinism(seed=None)
-        # compare TorchScript and PyTorch results
+        # compare TensorRT and PyTorch results
         for r1, r2 in zip(torch_out, trt_out):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)